pkg/hub/ratelimiting.go

```diff
@@ -37,8 +37,9 @@ type RateLimits struct {
 }
 
 var (
-	first  = "https://auth.docker.io/token?service=registry.docker.io&scope=repository:ratelimitpreview/test:pull"
-	second = "https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest"
+	first        = "https://auth.docker.io/token?service=registry.docker.io&scope=repository:ratelimitpreview/test:pull"
+	second       = "https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest"
+	defaultValue = -1
```

Comment:
nit: move this as a const?

Reply:
Looks not possible to have a const:

```
# github.com/docker/hub-tool/pkg/hub
pkg/hub/ratelimiting.go:77:21: cannot take the address of defaultValue
pkg/hub/ratelimiting.go:78:21: cannot take the address of defaultValue
pkg/hub/ratelimiting.go:79:21: cannot take the address of defaultValue
pkg/hub/ratelimiting.go:80:21: cannot take the address of defaultValue
```
:sweat_smile: Is that fine to keep it like that ?

Reply:
Oh right :facepalm: